### PR TITLE
fix(@desktop/browser): Unable to sign message from simepledapp.eth

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
### What does the PR do

Removes the logic to convert a password into a hashPassword which was causing the issue

### Affected areas

Browser

### Screenshot of functionality

https://user-images.githubusercontent.com/60327365/161793990-f0c9e73e-69c9-4b70-9e98-33ddfbb25a83.mov

### CODE REVIEW

File: src/app_service/service/provider/async_tasks.nim [hunk 1]
```diff
@@ -11,13 +11,10 @@ const postMessageTask: Task = proc(argEncoded: string) {.gcsafe, nimcall.} =
 
   let jsonMessage = arg.message.parseJson
   let password = jsonMessage{"payload"}{"password"}.getStr
```
Comment (alaibe) [inline src/app_service/service/provider/async_tasks.nim:13]: unused variable I think

File: src/app_service/service/provider/async_tasks.nim [hunk 2]
```diff
@@ -11,13 +11,10 @@ const postMessageTask: Task = proc(argEncoded: string) {.gcsafe, nimcall.} =
 
   let jsonMessage = arg.message.parseJson
```
Comment (alaibe) [inline src/app_service/service/provider/async_tasks.nim:12]: do u need to parse it? you can just pass the message to the provider request

**Review by alaibe — APPROVED** [review]

File: src/app_service/service/provider/async_tasks.nim (hunk 1 above)
Comment (Khushboo-dev-cpp) [reply src/app_service/service/provider/async_tasks.nim]: yep, removed

File: src/app_service/service/provider/async_tasks.nim (hunk 2 above)
Comment (Khushboo-dev-cpp) [reply src/app_service/service/provider/async_tasks.nim]: changed :)

**Review by anastasiyaig — APPROVED** [review]

**Comment by anastasiyaig** [comment]
https://ethstatus.testrail.net/index.php?/runs/view/8908

**Pull request merged**